test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -1161,6 +1161,15 @@ def forward(self, x):
         x = torch.tensor([0.9920, -1.0362, -1.5000, 3.5000], requires_grad=True)
         self.run_test(Round(), x)
 
+    @skipIfUnsupportedMinOpsetVersion(11)
+    def test_det(self):
+        class Det(torch.nn.Module):
+            def forward(self, x):
+                return torch.det(x)
+
+        x = torch.randn(2, 3, 5, 5, requires_grad=True)
```

Comment:
Can we add another test case where input shape is (5, 5)? That covers the case where output is scalar.